### PR TITLE
new project categories: Data, infrastructure and web apps

### PR DESCRIPTION
**Pull request by OriHoch**
Feel free to treat this PR as an RFC to discuss the categories / projects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +46,21 @@ A curated list of awesome Open Source Software made in Israel, and by Israelis.
 * [cron-to-quartz](https://github.com/lirantal/cron-to-quartz) - Node.js library to convert unix or linux CRON syntax to Quartz Scheduler
 * [licenseye](https://github.com/lirantal/licenseye) - Node.js CLI tool to visualize an aggregate list of your dependencies' licenses
 
+### Data processing, aggregations, statistics and visualizations
+
+* [Israel Crime Data](https://github.com/orihoch/israel-crime-data) - traffic violations and police events, per city/neighborhood (coded), processed from source XLS files to static datapackages, DB and Elasticsearch + Kibana for visualizations
+* [National Library of Israel](https://github.com/orihoch/nli-data-pipelines) - data scraped from the National Library of Israel API, mostly photos and related metadata. Downloads photos to Google Storage, Elasticsearch + Kibana for visualizations.
+* [Knesset Data](https://github.com/hasadna/knesset-data-pipelines) - redesign / reimplementation of [Open Knesset](github.com/hasadna/Open-Knesset) - focusing on getting quality non-biased data about the Knesset.
```
Comment (lirantal) [inline README.md]: - missing the leading https://

File: README.md [hunk 2]
```diff
@@ -43,6 +46,21 @@ A curated list of awesome Open Source Software made in Israel, and by Israelis.
...
+
+### Infrastructure
+
+* [The Midburn Kubernetes Environment](https://github.com/Midburn/midburn-k8s) - Docker/Kubernetes infrastructure as code repository which provides the infrastructure for [Midburn](https://midburn.org/) - includes some custom webapps, different DBs, a Drupal installation and other related infra. components.
+* [Beit Hatfutsot Kubernetes Environment](https://github.com/beit-hatfutsot/mojp-k8s) - Docker/Kubernetes infrastructure as code repository which provides the infrastructure for parts of [The Museum Of The Jewish People](https://www.bh.org.il/) - includes frontend/backend webapps, data processing pipelines with Elasticsearch
+
+### Web Apps
+
+* [Spark](https://github.com/Midburn/spark) - (Not Apache Spark!) A NodeJS web-app that provides API and other central services for the Midburn organization.
```
Comment (lirantal) [inline README.md]: - change NodeJS to Node.js

**Comment by lirantal** [comment]
@OriHoch Thanks for adding and organizing the projects by categories, I think it makes sense.

Some projects are low-profile and I don't think fitting for the repo but since we're still small and my bet is that the local community is completely unaware of all these projects I am inclined to merge so we can build a collection, and will later thin stuff out.

**Comment by benjamingr** [comment]
@OriHoch can you add hasadna under "contributing"? (In a separate PR)

**Pull request closed without merging**